src/mbgl/map/transform.cpp

```diff
@@ -46,6 +46,10 @@ Transform::Transform(MapObserver& observer_,
 #pragma mark - Map View
 
 bool Transform::resize(const Size size) {
+    if (!size) {
+        return false;
```

Comment:
I think at one point the return value was used to check if the size was _changing_, but it looks like this may be used only in tests. If so, yes, `void` and throw on invalid input.